libs/block-template-custom-element/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "build": "block-scripts build",
-    "codegen": "rimraf ./src/types && block-scripts codegen && yarn format && yarn fix:eslint",
+    "codegen": "set -o pipefail; block-scripts codegen | xargs -I {} yarn prettier --write {}",
```

Comment:
A bit unfortunate that this command will no longer work on Windows terminal but think it's fine to require use of WSL for now

Reply:
I guess you mean `cmd` or powershell?

Regardless this was broken on plain `sh` as you pointed out to me offline 😭, so I've compromised back to `&&` and just formatting everything. It's unpleasant and causes side-effects but it's very easy to locate, understand, and disable for anyone who is bothered by that behaviour.

Reply:
Seems like a fine compromise for now, can revisit if needed